src/main/java/com/zipcodewilmington/looplabs/IntegerDuplicateDeleter.java

```diff
@@ -1,8 +1,139 @@
 package com.zipcodewilmington.looplabs;
 
+import java.util.Arrays;
+
+import java.lang.reflect.Array;
+
 /**
  * Created by leon on 1/29/18.
  * @ATTENTION_TO_STUDENTS You are forbidden from modifying the signature of this class.
  */
 public final class IntegerDuplicateDeleter extends DuplicateDeleter<Integer> {
+
+
+    public IntegerDuplicateDeleter(Integer[] intArray) {
+        super(intArray);
+    }
+
+    public static void main (String [] args)
+    {
+        Integer[] tester={1, 2, 2, 3, 3, 4, 6, 7, 8, 8, 9};
+        int count = counter(tester,3);
+        System.out.println(count);
```

Comment:
don't write tests in your main class, or if you do, don't commit them. Useless code shouldn't make it to your final projects like this.